index.src.html

```diff
@@ -429,12 +429,30 @@ <h3 id="remote-device">
     Does this specification allow an origin access to other devices?
   </h3>
 
-    Accessing other devices, both via network connections and via
-      direct connection to the user's machine (e.g. via Bluetooth,
-      NFC, or USB), could expose vulnerabilities - some of
-      these devices were not created with web connectivity in mind and may be inadequately
-      hardened against malicious input, or with the use on the web.
-
+  Accessing other devices, both via network connections and via
+  direct connection to the user's machine (e.g. via Bluetooth,
+  NFC, or USB), could expose vulnerabilities - some of
+  these devices were not created with web connectivity in mind and may be inadequately
+  hardened against malicious input, or with the use on the web.
+
+  Exposing other devices on a user’s local network also has significant privacy
+  risk:
+  *   If two user agents have the same devices on their local network, an
+      attacker may infer that the two user agents are running on the same host
+      or are being used by two separate users who are in the same physical
+      location.
+  *   Enumerating the devices on a user’s local network provides significant
+      entropy that an attacker may use to fingerprint the user agent.
+  *   If the specification exposes persistent or long lived identifiers of
+      local network devices, that provides attackers with a way to track a user
+      over time even if a user takes steps to prevent such tracking (e.g.
+      clearing cookies and other stateful tracking mechanisms).
+  *   Direct connections might be also be used to bypass security checks that
+      other APIs would provide. For example, attackers used the WebUSB API to
+      access others sites' crendentials on a hardware security, bypassing
+      same-origin checks in an early U2F API. [[ YUBIKEY-ATTACK ]]
```

Comment:
We should include an explicit reference to WebUSB here. It would serve as an example of how a spec may interact with things in unexpected ways.

Reply:
Will do.

Reply:
See 
https://github.com/w3ctag/security-questionnaire/pull/41/commits/7850ddae20ceb2d5beef8093f2916dd621a2485b.

Reply:
The WebUSB spec talks more about mitigations so I think that referencing it in the example mitigations section is a bit more direct?  Unless you wanted to make the example be something like:

>   *   Direct connections might be also be used to bypass security checks that
>       other APIs would provide. For example, attackers used the WebUSB API to
>       access others sites' crendentials on a hardware security, bypassing
>       same-origin checks in an early U2F API. [[YUBIKEY-ATTACK]] This is despite
>       the specification setting forth a combination of mitigations based on 
>       user mediation / prompting, secure origins, and feature policy.

?

My inclination is to leave the Yubikey example and mitigations contained by the WebUSB spec separate.

Reply:
Yes something like that. The point is to say 'you've got webusb so your threat model is different than when webusb was not in your browser'